index.html

```diff
@@ -0,0 +1,99 @@
+                <div class="banner_text">일상의 모든 물건을<br>
+                    거래해 보세요</div>
+                <div class="looking"><a class="link_color" href="/items">구경하러 가기</a></div>
+            </div>
+            <img src="./Img_home_top.png">
+        </div>
+    </div>
+    <div>
+        <div class="main">
+            <div class="main_in">
+                <img src="./Frame 2608833.png">
+                <div class="items_first">
+                    <div class="items_first1">Hot Item</div>
+                    <div class="items_first_text">
+                        <div class="items_first2">인기 상품을<br>
+                        확인해보세요</div>
+                        <div class="items_first3">가장 HOT한 중고거래 물품을<br>
+                        판다 마켓에서 확인해 보세요</div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="main">
+            <div class="main_in">
+                <div class="items_second">
+                    <div class="items_second1">Search</div>
+                    <div class="items_second_text">
+                        <div class="items_second2">구매를 원하는<br>
+                            상품을 검색하세요</div>
+                        <div class="items_second3">구매하고 싶은 물품은 검색해서<br>
+                            쉽게 찾아보세요</div>
+                    </div>
+                </div>
+                <img src="./Img_home_02.png">
+            </div>
+        </div>
+        <div class="main">
+            <div class="main_in">
+                <img src="./Img_home_03.png">
+                <div class="items_third">
+                    <div class="items_third1">Register</div>
+                    <div class="items_third_text">
+                        <div class="items_third2"> 판매를 원하는<br>
+                            상품을 등록하세요</div>
+                        <div class="items_third3">어떤 물건이든 판매하고 싶은 상품을<br>
+                            쉽게 등록하세요</div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="main_bottom">
+        <div class="content_bottom">
+            <div class="text_bottom">믿을 수 있는<br>
+            판다마켓 중고 거래</div>
+            <div><img src="./Img_home_bottom.png"></div>
+        </div>
+    </div>
+    <footer>
+        <div class="footer_contents">
+                <p class="footer_logo">©codeit - 2024</p>
+            <div class="footer_FAQ">
+                <p><a class="footer_FAQ_rink" href="/privacy">Privacy Policy</a></p>
+                <p><a class="footer_FAQ_rink" href="/faq">FAQ</a></p>
+            </div>
+            <div class="footer_sns">
+                <a href="https://www.facebook.com/"><img src="./ic_facebook.png"></a>
+                <a href="https://x.com/"><img src="/ic_twitter.png"></a>
+                <a href="https://www.youtube.com/"><img src="/ic_youtube.png"></a>
+                <a href="https://www.instagram.com/"><img src="/ic_instagram.png"></a>
+            </div>
```

Comment:
💊 제안  
이 영역은 로고들의 모음이라고 할 수 있습니다. 관련된 요소들끼리 묶인 목록이니 의미록적인 마크업을 작성한다면 list 관련 태그로 작성하시는 것을 추천드립니다.

```suggestion
            <ul class="footer_sns">
                <li><a href="https://www.facebook.com/"><img src="./ic_facebook.png"></a></li>
                <li><a href="https://x.com/"><img src="/ic_twitter.png"></a></li>
                <li><a href="https://www.youtube.com/"><img src="/ic_youtube.png"></a></li>
                <li><a href="https://www.instagram.com/"><img src="/ic_instagram.png"></a></li>
            </ul>
```